sass/themes/default.sass

```diff
@@ -19,12 +19,8 @@
     color: #666
 
   .markdown-body
-    font-family: 'A-OTF UD Shin Go Pro', 'Noto Sans Japanese', 'Kozuka Gothic Pr6N', 'Kozuka Gothic Pro', 'Meiryo UI', 'Meiryo', 'Helvetica Neue', Helvetica, 'Segoe UI', Arial, freesans, sans-serif, 'Apple Color Emoji', 'Segoe UI Emoji', 'Segoe UI Symbol'
     font-size: 29px
 
-    tt, code
-      font-family: 'SourceHanCodeJP-Normal', 'Ricty', Consolas, "Liberation Mono", Menlo, Courier, monospace
```

Comment:
Removed `font-family` definition from default theme. These definition would be delegated to [github-markdown-css](https://github.com/sindresorhus/github-markdown-css).